tests/std/tests/VSO_0180469_ptr_cat/test.cpp

```diff
@@ -186,6 +186,17 @@ void ptr_cat_test_cases() {
     test_ptr_cat<2, volatile int*, const volatile int*>();
     test_ptr_cat<2, const volatile int*, const volatile int*>();
 
+    // Pointer to pointer should work
+    test_ptr_cat<2, int**, int**>();
+    test_ptr_cat<2, pod_struct**, pod_struct**>();
+    test_ptr_cat<2, trivially_copyable_struct**, trivially_copyable_struct**>();
+    test_ptr_cat<2, custom_copy_struct**, custom_copy_struct**>();
+
+    // Pointer to pointer of different types not

```

Comment:
This comment needs a bit of grammar work. (Test code comments don't have to be perfectly phrased - they can be sentence fragments, but this one is too fragmented.)

Reply:
Done